spec/features/content_change_requests_spec.rb

```diff
@@ -83,6 +87,7 @@ def user_makes_a_content_change_request(details)
 
     fill_in "Title of request", with: details[:title] unless details[:title].nil?
     select details[:reason_for_change], from: "What’s the reason for the request?" unless details[:reason_for_change].nil?
+    select details[:subject_area], from: "What’s the subject area?" unless details[:subject_area].nil?
```

Comment:
Both of these `select`'s `from:` values [can refer to the ID/name of the select](https://www.rubydoc.info/github/jnicklas/capybara/master/Capybara/Node/Actions#select-instance_method), rather than the text (makes it easier to rename the label of the field). So I think this would work...?

```suggestion
    select details[:reason_for_change], from: "reason_for_change" unless details[:reason_for_change].nil?
    select details[:subject_area], from: "subject_area" unless details[:subject_area].nil?
```

Reply:
This doesn't work unfortunately